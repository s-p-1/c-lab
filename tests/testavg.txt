disable_output
A1=AVG(A2:ZZZ999)
A2=AVG(A3:ZZZ999)
A3=AVG(A4:ZZZ999)
A4=AVG(A5:ZZZ999)
A5=AVG(A6:ZZZ999)
A6=AVG(A7:ZZZ999)
A7=AVG(A8:ZZZ999)
A8=AVG(A9:ZZZ999)
A9=AVG(A10:ZZZ999)
A10=AVG(A11:ZZZ999)
A11=AVG(A12:ZZZ999)
A12=AVG(A13:ZZZ999)
A13=AVG(A14:ZZZ999)
A14=AVG(A15:ZZZ999)
A15=AVG(A16:ZZZ999)
A16=AVG(A17:ZZZ999)
A17=AVG(A18:ZZZ999)
A18=AVG(A19:ZZZ999)
A19=AVG(A20:ZZZ999)
A20=AVG(A21:ZZZ999)
A21=AVG(A22:ZZZ999)
A22=AVG(A23:ZZZ999)
A23=AVG(A24:ZZZ999)
A24=AVG(A25:ZZZ999)
A25=AVG(A26:ZZZ999)
A26=AVG(A27:ZZZ999)
A27=AVG(A28:ZZZ999)
A28=AVG(A29:ZZZ999)
A29=AVG(A30:ZZZ999)
A30=AVG(A31:ZZZ999)
A31=AVG(A32:ZZZ999)
A32=AVG(A33:ZZZ999)
A33=AVG(A34:ZZZ999)
A34=AVG(A35:ZZZ999)
A35=AVG(A36:ZZZ999)
A36=AVG(A37:ZZZ999)
A37=AVG(A38:ZZZ999)
A38=AVG(A39:ZZZ999)
A39=AVG(A40:ZZZ999)
A40=AVG(A41:ZZZ999)
A41=AVG(A42:ZZZ999)
A42=AVG(A43:ZZZ999)
A43=AVG(A44:ZZZ999)
A44=AVG(A45:ZZZ999)
A45=AVG(A46:ZZZ999)
A46=AVG(A47:ZZZ999)
A47=AVG(A48:ZZZ999)
A48=AVG(A49:ZZZ999)
A49=AVG(A50:ZZZ999)
A50=AVG(A51:ZZZ999)
A51=AVG(A52:ZZZ999)
A52=AVG(A53:ZZZ999)
A53=AVG(A54:ZZZ999)
A54=AVG(A55:ZZZ999)
A55=AVG(A56:ZZZ999)
A56=AVG(A57:ZZZ999)
A57=AVG(A58:ZZZ999)
A58=AVG(A59:ZZZ999)
A59=AVG(A60:ZZZ999)
A60=AVG(A61:ZZZ999)
A61=AVG(A62:ZZZ999)
A62=AVG(A63:ZZZ999)
A63=AVG(A64:ZZZ999)
A64=AVG(A65:ZZZ999)
A65=AVG(A66:ZZZ999)
A66=AVG(A67:ZZZ999)
A67=AVG(A68:ZZZ999)
A68=AVG(A69:ZZZ999)
A69=AVG(A70:ZZZ999)
A70=AVG(A71:ZZZ999)
A71=AVG(A72:ZZZ999)
A72=AVG(A73:ZZZ999)
A73=AVG(A74:ZZZ999)
A74=AVG(A75:ZZZ999)
A75=AVG(A76:ZZZ999)
A76=AVG(A77:ZZZ999)
A77=AVG(A78:ZZZ999)
A78=AVG(A79:ZZZ999)
A79=AVG(A80:ZZZ999)
A80=AVG(A81:ZZZ999)
A81=AVG(A82:ZZZ999)
A82=AVG(A83:ZZZ999)
A83=AVG(A84:ZZZ999)
A84=AVG(A85:ZZZ999)
A85=AVG(A86:ZZZ999)
A86=AVG(A87:ZZZ999)
A87=AVG(A88:ZZZ999)
A88=AVG(A89:ZZZ999)
A89=AVG(A90:ZZZ999)
A90=AVG(A91:ZZZ999)
A91=AVG(A92:ZZZ999)
A92=AVG(A93:ZZZ999)
A93=AVG(A94:ZZZ999)
A94=AVG(A95:ZZZ999)
A95=AVG(A96:ZZZ999)
A96=AVG(A97:ZZZ999)
A97=AVG(A98:ZZZ999)
A98=AVG(A99:ZZZ999)
A99=AVG(A100:ZZZ999)
A99=100
enable_output
scroll_to A90
q

