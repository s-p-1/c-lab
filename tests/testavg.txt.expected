              A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) > [0.0] (ok) > [0.2] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
 90           0           0           0           0           0           0           0           0           0           0
 91           0           0           0           0           0           0           0           0           0           0
 92           0           0           0           0           0           0           0           0           0           0
 93           0           0           0           0           0           0           0           0           0           0
 94           0           0           0           0           0           0           0           0           0           0
 95           0           0           0           0           0           0           0           0           0           0
 96           0           0           0           0           0           0           0           0           0           0
 97           0           0           0           0           0           0           0           0           0           0
 98           0           0           0           0           0           0           0           0           0           0
 99         100           0           0           0           0           0           0           0           0           0
[0.0] (ok) > 